README.md

```diff
@@ -1,27 +1,44 @@
 <!-- ix-docs-ignore -->
+<!-- Badges -->
 ![imgix logo](https://assets.imgix.net/sdk-imgix-logo.svg)
 
 `imgix-go` is a client library for generating image URLs with [imgix](https://www.imgix.com/).
 
 ![Version](https://badge.fury.io/gh/imgix%2Fimgix-go.svg)
-[![Build Status](https://travis-ci.org/imgix/imgix-go.svg?branch=main)](https://travis-ci.org/parkr/imgix-go)
+[![Build Status](https://travis-ci.org/imgix/imgix-go.svg?branch=main)](https://travis-ci.org/imgix/imgix-go)
 [![Godoc](https://godoc.org/github.com/imgix/imgix-go?status.svg)](https://godoc.org/github.com/imgix/imgix-go)
 [![License](https://img.shields.io/github/license/imgix/imgix-go)](https://github.com/imgix/imgix-go/blob/main/LICENSE)
 
 ---
 <!-- /ix-docs-ignore -->
 
+<!-- Table of Contents -->
 - [Installation](#installation)
 - [Usage](#usage)
+- [Signed URLs](#signed-urls)
+- [Srcset Generation](#srcset-generation)
+  - [Fixed-Width Images](#fixed-width-images)
+    - [Variable Quality](#variable-quality)
+  - [Fluid-Width Images](#fluid-width-images)
+    - [Custom Widths](#custom-widths)
+    - [Width Ranges](#width-ranges)
+    - [Width Tolerance](#width-tolerance)
+    - [Explore Target Widths](#explore-target-widths)
+- [The `ixlib` Parameter](#the-ixlib-parameter)
+- [Testing](#testing)
 
+<!-- Installation Instructions -->
 ## Installation
 
 ```bash
 go get github.com/imgix/imgix-go
 ```
 
+<!-- Usage Instructions -->
 ## Usage
 
+To begin creating imgix URLs, import the imgix library and create a URL builder. The URL builder can be reused to create URLs for any images on the domains it is provided.
```

Comment:
I think this should probably be "domain" rather than "domains"?

```suggestion
To begin creating imgix URLs, import the imgix library and create a URL builder. The URL builder can be reused to create URLs for any images on the domain it is provided.
```